ci/provisioning/post_provision_config_nodes_EL_8.sh

```diff
@@ -49,6 +49,31 @@ distro_custom() {
 
     dnf config-manager --disable powertools
 
+    # Mellanox OFED hack
+    if ls -d /usr/mpi/gcc/openmpi-*; then
+        version="$(rpm -q --qf %{version} openmpi)"
```

Comment:
(lint) This { is literal. Check expression (missing ;/\n?) or quote it. [SC1083]
(lint) This } is literal. Check expression (missing ;/\n?) or quote it. [SC1083]